### DISCUSSION TITLE
422 Unprocessable Entity，Multiple fileupload

### DISCUSSION THREAD

**Discussion by mkmkmkmkmkmkmkmkmkmkmk** [fastapi/fastapi#10987] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
const fd = new FormData()
      if (this.fileList.length > 0) {
        this.fileList.forEach(function(file, i) {
          fd.append('file', file.raw)
          console.log(file.raw)
        })
      }
      console.log(fd.getAll('file'))
      axios.post('http://127.0.0.1:8000/v1/uploadMultiFiles', fd, this.config).then((response) => {
        this.$message({
          showClose: true,
          message: '上传服务器成功！',
          type: 'success'
        })
      }).catch(error => {
        console.log(error)
        this.$message({
          showClose: true,
          message: '请求出现异常！',
          type: 'error'
        })
      })

@app.post("/v1/uploadMultiFiles/")
# 定义路径操作函数，当接口被访问将调用该函数
async def DealMlutiUpload(list:List[UploadFile]):
    try:

        # for file in  fileList:
        #     filename = file.filename
        #     res = await file.read()
        #     path = "res/uploadrename/" + filename
        #     with open(path, "wb") as f:
        #         f.write(res)
        # results = {"message": "上传成功：{}".format(filename)}
        results = {"message": "上传成功"}
        return  results
    except Exception as e:
        print("异常信息：", e)
        raise HTTPException(status_code=500, detail="上传文件失败，服务器端异常！")
```

### Description

1. My front-end code sends the request code like this
![image](https://github.com/tiangolo/fastapi/assets/90512746/3f77127d-fc63-490b-b7d4-4b354c2eff5d)
2. The backend code looks like this：
![image](https://github.com/tiangolo/fastapi/assets/90512746/e5c3ea0f-6e00-4cdd-8814-a42ced5186c8)
3. console.log(fd.getAll("file"))输出如下：
![image](https://github.com/tiangolo/fastapi/assets/90512746/98837e30-80e2-4d7a-8cd6-13e294f2cdfe)
4.Why does the 422 error occur when accessing this interface '/v1/uploadFiles/'? How should I receive this fd request data on the backend? How should I modify it？
![image](https://github.com/tiangolo/fastapi/assets/90512746/3902f32b-670d-4697-8717-8f46ba895676)

### Operating System

Windows

### FastAPI Version

0.19.0

### Pydantic Version

1.10.8

### Python Version

3.11.5

**Comment by Kludex** [comment]
What's the response body on the client side?

**Comment by YuriiMotov** [comment]
Let's close this for now to keep the discussions clean and organized.
If you’ve found a solution, please share it here.
If you still need help, feel free to let us know.
